co-simulation/lib/mosaic-objects/src/test/java/org/eclipse/mosaic/lib/objects/detector/DetectedObjectTest.java

```diff
@@ -218,7 +218,7 @@ public void testEquals() {
                 0.6,
                 null,
                 null,
-                null,
+                100,
```

Comment:
is this correct? why is it not null but 100? does it matter?

Reply:
unit testing passed, shouldn't be a problem